src/screens/ContactScreen/elements/ItemUser.js

```diff
@@ -50,8 +52,12 @@ const ItemUser = ({photo, username, followed, onPress, userid}) => {
         </View>
         <View style={styles.containerButton}>
           <Pressable style={styles.followAction(32, 32)}>
-            <Animated.View style={{position: 'absolute', opacity: followIconFadeAnimation}}>
-              <MemoIc_Checklist width={32} height={32} />
+            <Animated.View style={{position: 'absolute', top: 4, opacity: followIconFadeAnimation}}>
+              <IconChecklist
+                width={32}
+                height={32}
+                color={isAnon ? COLORS.anon_primary : COLORS.signed_primary}
+              />
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `Animated.View` component is used here but there's no animation happening. If you're planning to add animations later, it's fine. Otherwise, consider replacing `Animated.View` with a regular `View` component for simplicity.
```diff
- <Animated.View style={{position: 'absolute', top: 4, opacity: followIconFadeAnimation}}>
+ <View style={{position: 'absolute', top: 4}}>
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->